### DISCUSSION TITLE
MC Shortcuts not available on Mac Boot Up / Error Message

### DISCUSSION THREAD

**Discussion by robfoll** [MonitorControl/MonitorControl#1392] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

On almost every startup, MC fails to be recognised in Pricay / Accessibility despite being previously selected and switched on. Happens approx 80% of reboots. Sometimes rebooting twice fixes the problem

Shortcuts not available
You need to enable MonitorControl in
System Preferences > Security and
Privacy > Accessibility for the keyboard shortcuts to work

### Steps to reproduce

Reboot Mac, message appears after system has loaded

### Expected behavior

Machine to reboot with no messages relating to MC

### Anything else?

<img width="638" alt="Screenshot 2023-06-15 at 12 46 10" src="https://github.com/MonitorControl/MonitorControl/assets/125624165/2c9d4ddf-57d2-444f-8457-f9ff7c0b2d8e">

### Environment Information (please complete the following information)

```markdown
- macOS version:13.4 (22F66)
- Mac model: MacBook Air M2
- MonitorControl version: 4.1
- Monitor(s):0 just on laptop currently
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
You'll probably need to reset the privacy database in Terminal: `tccutil reset Accessibility`
